main.py

```diff
@@ -60,6 +60,14 @@ def setup(self):
 
         self.load_all_evcs()
 
+    def get_evcs_by_svc_level(self) -> list:
+        """Get circuits sorted by desc service level and asc creation_time.
+
+        In the future, as more ops are offloaded it should be get from the DB.
```

Comment:
Ideally, in the future this should be offloaded to the DB, but considering the prior strategy when it was migrated to MongoDB, then it should be included in a next opportunity when more things are offloaded to the DB.